src/connectors/extends.js

```diff
@@ -0,0 +1,14 @@
+const vec3 = require('../math/vec3')
+const fromPointAxisNormal = require('./fromPointAxisNormal')
+
+/** Creates a new Connector, with the connection point moved in the direction of the axis
+ * @param {Number} distance the distance to extend the connector to
+ * @param {Connector} connector the connector to extend
+ * @returns {Connector} a normalized connector
+ */
+const extend = (distance, connector) => {
```

Comment:
I don't understand the need for this function. You can do the same via transform().
Therefore, I didn't create tests and commented out of the index.

Reply:
I think it is not quite the same thing, (the naming is ...odd but got no better idea right now), the connector point gets moved in the *direction* of the axis, not just transformed.
I think we should leave this in for now, add it to the list of things to double check in the cleanup pass if that is ok with you ? (We will have the ability to visually compare between implementations, so that will help)

Reply:
Ok. We can keep it. But I won’t create tests until we find a use case.